<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>genesis_mwin</class>
 <widget class="QMainWindow" name="genesis_mwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Genesis</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="central_vbox">
    <item>
     <widget class="QPushButton" name="shot_open_pb">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="asset_open_pb">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="shot_save_pb">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Save/New</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="asset_save_pb">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Save/New</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="asset_descriptor_le"/>
    </item>
    <item>
     <widget class="QLabel" name="asset_descriptor_lb">
      <property name="text">
       <string>Descriptor:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="shot_descriptor_le"/>
    </item>
    <item>
     <widget class="QLabel" name="shot_descriptor_lb">
      <property name="text">
       <string>Descriptor:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
